stripe/src/main/java/com/stripe/android/view/PaymentAuthWebViewActivityViewModel.kt

```diff
@@ -156,10 +126,4 @@ internal class PaymentAuthWebViewActivityViewModel(
             ) as T
         }
     }
-
-    private companion object {
-        private const val FIELD_CHALLENGE_URI = "challenge_uri"
-        private const val FIELD_ERROR_MESSAGE = "error_message"
-        private const val FIELD_ERROR_STACKTRACE = "error_stacktrace"
-    }
```

Comment:
please also include these fields in the removed params section in PR description